doc/doxygen-layout-user.xml

```diff
@@ -77,10 +77,10 @@
       <typedefs title=""/>
       <functions title=""/>
       <properties title="OpenSim Property, Connector, Output, Input Documentation"/>
-      <!-- <enums title=""/>
+      <enums title=""/>
       
       
-      <related title=""/>
+      <!-- <related title=""/>
```

Comment:
> I'm pretty sure this means that the enums appear at the very very bottom of the pages.

That is correct.

> Is this desirable?

Yes, as far as I can tell. Previous to this PR, enums were not being shown in the user doxygen. I, therefore, concluded that the author of doxygen-layout-user.xml established that users would not need to know about enums (though I disagree; see below). Thus, it did not seem reasonable to include the enums at the top of the page. For the joints, the `getCoordinate()` and `updCoordinate()` methods appear at the bottom of the "Member Function Documentation" section; the "Member Enumeration Documentation" section follows immediately thereafter, which is ideal.

> Should they also be under the declaration section (<memberdecl>)?

I don't think it's necessary to show the enum documentation anywhere but where it currently appears.

> Have you checked to see how this affects the documentation of any other enums?

Yes. One example is Thelen2003Muscle, whose `printCurveToCSVFile()` method takes a `CurveType` enum as its first argument. The table of possible `CurveType`s now appears at the bottom of the Thelen2003Muscle doxygen page and is hyperlinked from the first argument of `printCurveToCSVFile()`.

> Are there cryptic enums that should be hidden from users that are now shown?

Here is a list of all the documented enums:

![enums](https://cloud.githubusercontent.com/assets/4203505/18732499/9399d39c-8018-11e6-9890-664a3860d16b.png)

I think the following entries are useful: Coord, CurveType, MotionType, UnitType, ValueType; the remaining entries are harmless (the Property_Deprecated and Tool pages are unlikely to be visited by users anyway).